cpp/cmake/thirdparty/get_arrow.cmake

```diff
@@ -273,7 +273,10 @@ function(find_and_configure_arrow VERSION BUILD_STATIC ENABLE_S3 ENABLE_ORC ENAB
 
 endfunction()
 
-set(CUDF_VERSION_Arrow 8.0.0)
+set(CUDF_VERSION_Arrow
```

Comment:
We should wrap this in `if(NOT DEFINED CUDF_VERSION_Arrow)` or set CMP0126 to NEW ( https://cmake.org/cmake/help/latest/policy/CMP0126.html ). That way project that use cudf and pass `CUDF_VERSION_Arrow` as a non-cache variable will be obeyed.

Reply:
I'll wrap it in the `if(NOT DEFINED ...)` for now because I imagine setting the policy has wider implications that shouldn't be tackled in this PR.